betty/cropper/templates/image.js

```diff
@@ -34,7 +34,10 @@
   }
 
   w.picturefill = function(elements, forceRerender) {
-
+    // It is sometimes desirable to scroll without loading images as we go.
+    if (w.pauseBettyCroppyPicturefill) {
+      return;
```

Comment:
Might be confusing since ```picturefill``` is a function that's called and not a persistent object, but can this go under a namespace, so it's use would be ```window.picturefill.paused()``` where ```paused``` is a getter/setter. This way we avoid further polluting the already extremely polluted window.